scripts/proto_splitter/proto2mesh.py

```diff
@@ -0,0 +1,453 @@
+#!/usr/bin/env python3
+
+# Copyright 1996-2021 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author Simon Steinmann (https://github.com/Simon-Steinmann)
+#
+
+
+# Automatic PROTO to multi-file PROTO trimesh extractor
```

Comment:
it would be good to add here what is the difference between this and the `proto2mesh_multiThreaded.py` script and when a user should use one or the other script.

Reply:
I will actually remove this script from this PR as @Simon-Steinmann suggested he will actually merge it with the main script in a separate PR.